docs/developer/developer-mode-installation.md

```diff
@@ -58,16 +58,19 @@ You can install Harvester on an [RKE2](https://docs.rke2.io/) cluster using the
     sudo cp /etc/rancher/rke2/rke2.yaml ~/.kube/config
     sudo chown $(id -u):$(id -g) ~/.kube/config
     ```
+:::info note
+Wait until RKE2 will be fully up (all pods in `kube-system` namespace will be running) before continuing with next steps.
+:::
+
+:::info note
+If you are unable to locate the `kubectl` binary in `/usr/local/bin`, check `/var/lib/rancher/rke2/bin`.
```

Comment:
May you put two note in one `info note` block? Original render result is like following:

![Screenshot 2024-08-21 at 1 36 46 PM](https://github.com/user-attachments/assets/2c14aa1f-e1a6-49c7-8265-57e7df7cbd8a)

Reply:
@FrankYang0529 Thanks for reviewing. I'll clean up the wording and markup when I do my own review. Which doc versions should be updated?

Reply:
I think we can also apply this to v1.3 document. Thanks.